modules/extensions-support/src/test/java/org/mule/module/extension/internal/metadata/extension/MetadataExtension.java

```diff
@@ -10,11 +10,14 @@
 import org.mule.extension.api.annotation.Operations;
 import org.mule.extension.api.annotation.capability.Xml;
 import org.mule.extension.api.annotation.connector.Providers;
+import org.mule.extension.api.annotation.metadata.MetadataScope;
+import org.mule.module.extension.internal.metadata.extension.resolver.TestContentAndOutputResolverWithKeyResolver;
 
 @Extension(name = "Metadata")
-@Operations({MetadataOperations.class, MetadataFailureOperations.class})
+@Operations({MetadataOperations.class, MetadataFailureOperations.class, MetadataScopeOperations.class})
```

Comment:
I needed one operation class that wasn't annotated with @MetadataScope just for the sake of testing the inheritance. The name is probably not the best because it leads to some confusion.